src/main/java/org/team199/robot2022/commands/ExtendClimber.java

```diff
@@ -9,24 +9,23 @@
 import edu.wpi.first.wpilibj2.command.FunctionalCommand;
 import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
 
-public class ExtendClimber extends ParallelCommandGroup {
+public final class ExtendClimber extends ParallelCommandGroup {
```

Comment:
```suggestion
public class ExtendClimber extends ParallelCommandGroup {
```
Sorry, my bad. The `final` comment was meant to refer to the member variable not the class. The commands do not have to be `final`.

Reply:
Java throws an error that you're not allowed to use the 'final' modifier on the "public (Retract|Extend)Climber(Climber climber)"

Reply:
I was talking about the line `private Climber climber;` and changing it to `private final Climber climber;` It looks like it was deleted in a previous commit, so no further changes are necessary.